easybuild/easyconfigs/n/nsync/nsync-1.29.2-GCCcore-13.2.0.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'CMakeNinja'
+
+name = 'nsync'
+version = '1.29.2'
+
+homepage = 'https://github.com/google/nsync'
+description = """nsync is a C library that exports various synchronization primitives, such as mutexes"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/google/nsync/archive/v%(version)s/']
```

Looks like the folder is to much:
```suggestion
source_urls = ['https://github.com/google/nsync/archive']
```